NuGet.config

```diff
@@ -3,5 +3,9 @@
   <packageSources>
     <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+    <add key="BuildTools" value="https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
```

Comment:
Just need the msbuild packages. 
I am assuming some time in the future, we will again start moving the packages to the AspNetCore feed. So all the other feeds will be removed once we start getting new packages into our feed.